README.md

```diff
@@ -708,7 +717,7 @@ the following major changes to the residential modeling codebase:
   process was moved to [pipeline/00-ingest.R](pipeline/00-ingest.R),
   while the process to [finalize model
   values](https://gitlab.com/ccao-data-science---modeling/processes/finalize_model_values)
-  was moved to [pipeline/06-export.R](pipeline/06-export.R).
+  was moved to [pipeline/06-export.R](pipeline/07-export.R).
```

Comment:
```suggestion
  was moved to [pipeline/07-export.R](pipeline/07-export.R).
```

Reply:
Done in 7458babd12d1612b6f900c001672524ef63bf7c2.